activerecord/lib/active_record.rb

```diff
@@ -267,12 +267,13 @@ def self.legacy_connection_handling=(_)
 
   def self.global_thread_pool_async_query_executor # :nodoc:
     concurrency = global_executor_concurrency || 4
+    min_threads = global_executor_min_threads || 0
```

Comment:
For the executor I don't think it makes much sense to have a `min_threads`, if they are idle they don't cost much, so we might as well use a fixed thread pool with a fixed number of threads.